docs/pages/docs/core-concepts/remote-caching.mdx

```diff
@@ -9,13 +9,23 @@ import { Tabs, Tab } from "../../../components/Tabs";
 
 # Remote Caching
 
-A major key to Turborepo's speed is that it is both lazy and efficient—it does the least amount of work possible and it tries to never redo work that's already been done before.
+Turborepo's [task cache](/docs/core-concepts/caching) can save a lot of time by never doing the same work twice.
 
-By default, Turborepo caches the outputs of your tasks on your machine's local file system. We call this "single-player mode." But what if you could take advantage of the computational work done by your teammates or your CI by sharing a single cache across machines? In other words, what if you could turn on "co-op multiplayer mode", or a "Dropbox" for your Turborepo cache?
+But there's an issue - the **cache is local to your machine**. When you're working with a CI, this can result in a lot of duplicated work:
 
-> Remote Caching has entered the chat.
+![](../../../public/images/docs/local-caching.png)
 
-Turborepo can use a technique known as Remote Caching to share cache artifacts across machines for an additional speed boost. This is possible because Turborepo uses content addressable storage.
+Each environment - your machine, your colleague's machine, your CI - is having to **run `build` on the same source files**. That's because each environment has _its own local cache_.
```

Comment:
```suggestion
Each environment - your machine, your colleague's machine, your CI - has to **run `build` on the same source files**. That's because each environment has _its own local cache_.
```